src/doc/data_into_visit/CurveFormat.rst

```diff
@@ -0,0 +1,18 @@
+.. _Data_Into_VisIt_Curve:
+
+The Curve file format
+=====================
+
+A curve file is a text file for specifying 1D curves.
+It consists of one or more comment lines with meta data about the curve followed by the coordinates.
+The comment lines start with either a `#` or `%` character.
+Comment lines consist of keyword / value pairs.
+Supported keywords are `CYCLE` and `TIME`.
+The last comment line contains the name of the curve.
+The points consists of the X-Y coordinates for each point, one per line.
+The X coordinates must be monotonically increasing.
```

Comment:
The monotonically increasing requirement isn't strictly true. We support curves specified in polar coordinates as well.  There is no difference in the reader either way.

Reply:
@biagas, How do we know they are in polar coordinates?